docs/getting-started.md

```diff
@@ -18,14 +20,14 @@ Use `get_hconfig` to create HConfig objects for both the running and intended co
 
 ```python
 # Define running and intended configurations as strings
-running_config_text = open("./tests/fixtures/running_config.conf").read()
-generated_config_text = open("./tests/fixtures/generated_config.conf").read()
-
+>>> running_config_text = load_device_config("./tests/fixtures/running_config.conf")
+>>> generated_config_text = load_device_config("./tests/fixtures/remediation_config.conf")
+>>>
 
 # Create HConfig objects for running and intended configurations
-running_config = get_hconfig(Platform.CISCO_IOS, running_config_text)
-generated_config = get_hconfig(Platform.CISCO_IOS, generated_config_text)
-
+>>> running_config = get_hconfig(Platform.CISCO_IOS, running_config_text)
```

Comment:
`get_hconfig()` can take a Path. Should we encourage the use of that in this case?

```
def get_hconfig(
    platform_or_driver: Union[Platform, HConfigDriverBase],
    config_raw: Union[Path, str] = "",
) -> HConfig:
```

Reply:
I don't have a strong opinion, but I believe `Path("./tests/fixtures/running_config.conf").read_text(encoding="utf-8")` is more work and prone to typos compared to `load_device_config("./tests/fixtures/running_config.conf")`. Using a function that calls `Path` under the hood also aligns with the DRY principle.